tests/contrib/rq/test_rq.py

```diff
@@ -127,9 +127,8 @@ def test_worker_class_job(queue):
     worker.work(burst=True)
 
 
-@pytest.mark.parametrize(
-    "distributed_tracing_enabled,worker_service_name", [(None, "custom-worker-service"), (False, None)]
-)
+@pytest.mark.parametrize("distributed_tracing_enabled", [False, None])
+@pytest.mark.parametrize("worker_service_name", [None, "custom-worker-service"])
```

Comment:
@Kyle-Verhoog this is why the new snapshots are there, this small change increases the matrix of tests. probably should be in a different PR, let me know if you want that.

Reply:
yeah can we do that real quick?